serverless.yaml

```diff
@@ -37,6 +37,7 @@ custom:
   enableDynamicHostName: ${env:ENABLE_DYNAMIC_HOST_NAME, 'false'}
   enablePrivateAPIGateway: ${env:ENABLE_PRIVATE_API_GATEWAY, 'true'}
   importPrivateAPIGateway: ${env:IMPORT_PRIVATE_API_GATEWAY, 'false'}
+  apiGatewayLogRoleManagedExternally: ${env:API_GATEWAY_LOG_ROLE_MANAGED_EXTERNALLY, 'true'}
```

Comment:
I am ok with this approach, but we are going to have to update the Codebuild script to override the default for a fresh install, correct?

Reply:
yes, the default value is used by v1.0.3 upgrade. for fresh install, we need to set it to false.